src/mson_to_json_schema.js

```diff
@@ -65,6 +65,9 @@ function convert(mson) {
                     case 'required':
                         schema.required.push(member.content.key.content);
                         break;
+                    case 'nullable':
+                    schema.properties[member.content.key.content].type = [schema.properties[member.content.key.content].type, 'null']
+                    break;
```

Comment:
Oops, sorry... Just notice that the indent seems not correct...

Reply:
No problem!
My editor will fix it automatically at the next time I edit.